.env.template

```diff
@@ -12,6 +12,13 @@ MINIO_URL=localhost:9001
 MINIO_ACCESS_KEY=minioadmin
 MINIO_SECRET_KEY=minioadmin
 
+# Custom Minio credentials (uncomment and set these for production use)
```

Comment:
Here I'd have a section like "Object storage" you can use either MINIO or S3. If you want to use S3, set these 3 variables, if you want to use MINIO set these 3 variables. Make it easy to uncomment as necessary. 

But also default to MinIO, so leave those un-commented, and the same as the docker compose file.